CHANGELOG.MD

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.4]
```

Comment:
Since there was not release between 2.1.3 and 2.1.4 we could merge this one on the 2.1.3, what do you think?